pkg/apis/configuration/v1/kongplugin_types.go

```diff
@@ -64,6 +65,13 @@ type KongPlugin struct {
 	// Only one of `config` or `configFrom` may be used in a KongPlugin, not both at once.
 	ConfigFrom *ConfigSource `json:"configFrom,omitempty"`
 
+	// ConfigPatches represents JSON patches to the configuration of the plugin.
+	// Each item means a JSON patch to add something in the configuration,
+	// where path is specified in `path` and value is in `valueFrom` referencing
+	// a key in a secret.
+	// Could only be specified when Config specified.
```

Comment:
Ditto: Please add a CEL rule that would prevent ConfigPatches from being set when there's no Config set.

Reply:
There is already one.

Reply:
👍 Sorry, I missed that.